src/control/lib/telemetry/counter.go

```diff
@@ -3,8 +3,8 @@
 //
 // SPDX-License-Identifier: BSD-2-Clause-Patent
 //
-//go:build linux && amd64
-// +build linux,amd64
+//go:build linux && (amd64 || arm64)
+// +build linux,amd64 linux,arm64
 
```

Comment:
Shouldn't this be adding one more other build tags, not both?

```suggestion
//go:build linux && amd64
// +build linux,amd64
//
//go:build linux && arm64
// +build linux,arm64

```

Reply:
use two go:build build failed. so change to follow:
//go:build linux && (amd64 || arm64)
// +build linux
// +build amd arm64